Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1625,17 +1629,13 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
         UIApplication.shared.sendAction(#selector(UIResponder.resignFirstResponder), to:nil, from:nil, for:nil)
         var actions: [[PhotonActionSheetItem]] = []
```

Comment:
Hmm, are these sections in the action sheet? I expected a non-nested array

Reply:
The nested array acts as sections that is correct. Its kind of a weird way of doing it. But the lists are pretty small.